src/message.rs

```diff
@@ -434,6 +435,18 @@ pub struct Message {
 
     /// `In-Reply-To` header value.
     pub(crate) in_reply_to: Option<String>,
+
+    /// `Supersedes` header value.
+    ///
+    /// It is not stored in the database.
```

Comment:
```suggestion
    /// It is only used for sending and not stored in the database.
```
To minimize astonishment when reading this